src/Spryker/Zed/Install/Business/Executable/ExecutableFactory.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Zed\Install\Business\Executable;
+
+use Spryker\Zed\Install\Business\Configuration\ConfigurationInterface;
+use Spryker\Zed\Install\Business\Executable\CommandLine\CommandLineExecutable;
+use Spryker\Zed\Install\Business\Stage\Section\Command\CommandInterface;
+
+class ExecutableFactory
```

Comment:
Mb we don't need additional factories and can put everything in a one?

Reply:
I don't see an easy way to do that. This factory is parametrized and is used to process dynamic values of `$command`, `$configuration` into `$executable` inside the `CommandRunner` model. Extracting it from `CommandRunnder` implies heavy refactoring of this class and most probably all the related tests.